run-rchk/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM rhub/ubuntu-rchk
+
+USER root
```

Comment:
Do you actually need to do this? don't docker containers run as root by default?

Reply:
That's because `rhub/ubuntu-rchk` switched to `docker` in the first place.

Reply:
Oh I see, ok